music.py

```diff
@@ -3,20 +3,19 @@
 import youtube_dl
 import os
 import re
-import json
 import asyncio
-import time
 
 from discord.utils import get
 from youtubesearchpython import Search
 
 COLOR = 0x00f0fa
-QUEUES = []
-music_list = []
+MUSIC_LIST = []
+
 
 async def join(ctx, client):
     global voice
```

Comment:
`global` służy do tego żeby móc zmienić całkowicie obiekt pod zmienną globalną. Nie ma zmiennej `voice`.
 Jeśli chcecie globalną `VOICE` to musicie ją wcześniej zadeklarować.